### PR TITLE
Fixes #16082: http_request_content_headers test try to get content from a no longer defined url

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/16082

### CODE REVIEW

File: tests/acceptance/30_generic_methods/network/http_request_content_headers.cf [hunk 1]
```diff
@@ -22,8 +22,8 @@ body common control
 bundle agent init
 {
   vars:
-    "existant_url"          string => "http://www.ncf.io";
-    "nonexistant_url"       string => "http://www.ncf.io/testurl";
+    "existant_url"          string => "http://www.rudder.io";
```
Comment (amousset) [inline tests/acceptance/30_generic_methods/network/http_request_content_headers.cf]: could you use repository.rudder.io instead?

**Comment by Fdall** [comment]
Commit modified

File: tests/acceptance/30_generic_methods/network/http_request_content_headers.cf (hunk 1 above)
Comment (Fdall) [reply tests/acceptance/30_generic_methods/network/http_request_content_headers.cf]: sure

**Comment by Fdall** [comment]
PR updated with a new commit

File: tests/acceptance/30_generic_methods/network/http_request_content_headers.cf [hunk 2]
```diff
@@ -22,8 +22,8 @@ body common control
 bundle agent init
 {
   vars:
-    "existant_url"          string => "http://www.ncf.io";
-    "nonexistant_url"       string => "http://www.ncf.io/testurl";
+    "existant_url"          string => "http://www.repository.rudder.io";
```
[the remark this answers is not in the document]
Comment (Fdall) [reply tests/acceptance/30_generic_methods/network/http_request_content_headers.cf]: The test does not pass with repository.rudder.io :/
Worked fine with rudder.io

[the remark this answers is not in the document]
Comment (amousset) [reply tests/acceptance/30_generic_methods/network/http_request_content_headers.cf]: you probably need to change the checked header

**Comment by Fdall** [comment]
Commit modified

**Comment by Fdall** [comment]
PR rebased

**Comment by Fdall** [comment]
PR rebased

**Pull request merged**